mne_bids/mne_bids.py

```diff
@@ -253,7 +253,7 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
     """
     # get MEASurement date from the data info
     meas_date = raw.info['meas_date']
-    if isinstance(meas_date, (np.ndarray, list)):
+    if isinstance(meas_date, (tuple):
```

Comment:
how is this working on your computer? :)

Reply:
you didn't close a paranthesis

Reply:
thanks @jasmainak